prisma/schema.prisma

```diff
@@ -38,4 +42,22 @@ model Device {
 
   updated_at DateTime @updatedAt
   created_at DateTime @default(now())
+  deleted_at DateTime?
+}
+
+model Cat {
+  id String @unique @default(cuid())
+
+  name        String @default("Cat")
+  description String @default("Meow meow. Meow meow? Meow!")
```

Comment:
Why are these set to defaults?

Reply:
Initial thought process was to have something to display on frontend. I suppose only an image should have a default

Reply:
On second thought, I guess we could just have a default on the client side.